org.uqbar.project.wollok.lib/src/wollok.wlk

```diff
@@ -492,6 +496,24 @@ package lang {
 		method ==(other) native
 	}
 	
+	class Dictionary {
+	
+		constructor() { }
+		method put(_key, _value) native
+		method get(_key) native
+		method size() = return self.values().size()
+		method isEmpty() = return self.size() == 0
+		method containsKey(_key) = return self.keys().contains(_key)
+		method containsValue(_value) = return self.values().contains(_value)
+		method remove(_key) native
+		method keys() native
+		method values() native
+		method forEach(closure) native
+		method newInstance() = return new Dictionary()
+		method clear() native
+		
+	}
```

Comment:
I know that it doesn't hurt anyone, but shortcut methods don't need the ```return``` keyword.